### PR TITLE
Propose Lexik Translation for translation mgmt.

### PR DESCRIPTION
**Pull request by gabiudrescu**
I felt the need to fill the empty space of recommended libraries and bundles :neckbeard:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Welcome to the SyliusGoose, which is the place you will find more information ab
 
 ### Bundles, components and libraries
 
-- :cry:
+- [Lexik Translation Manager](https://github.com/lexik/LexikTranslationBundle) - It imports all translations from YML files to a database, allowing operators to them on the fly.
```
Comment (bitbager) [inline README.md]: Can we add a colon after the author? 🙂

**Comment by gabiudrescu** [comment]
OK now?

maybe we should have a contribution guide line :)

**Comment by bitbager** [comment]
Perfect. The guide is also a nice idea, maybe we can discuss the standards on Slack with other friends 🙂

**Comment by bitbager** [comment]
I also like the idea of short description around packages!

**Pull request merged**